.github/workflows/swagger.yml

```diff
@@ -0,0 +1,35 @@
+name: Publish Swagger Specs
```

Comment:
This can be replaced with a simpler implementation, see https://hmcts-reform.slack.com/archives/CA4F2MAFR/p1680087457871489

Reply:
Will also accommodate this in PRs. Thank you.